README.md

```diff
@@ -63,118 +45,64 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [Alhassan Osman's Portfolio](https://flemton.github.io/my-portfolio/) <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**[My Portfolio](https://flemton.github.io/my-portfolio/)** is one of the most powerful tool in my software developer toolbox; it’s the easiest way of showing what I am truly capable of as a developer, and is a quick and simple way for recruiters and hiring managers to get an idea of what I can bring to their teams.
+**[Click here](https://flemton.github.io/my-portfolio/) for a [live](https://flemton.github.io/my-portfolio/) version to [my portfolio](https://flemton.github.io/my-portfolio/)**
 
 ## 🛠 Built With <a name="built-with"></a>
+- PostgreSQL
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+* SQL
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://expressjs.com/">Node.js</a></li>
   </ul>
```

Comment:
- [OPTIONAL] I would advise you to remove this because `Node.js` was not used in this project. For the **_Tech Stack_** section, you ought to list technologies that were used in the project. 👍 😎

Reply:
Dear @Gambit142, thank you for the great suggestions, I have applied them, cheers 🥂

Reply:
I am glad I could be of help 👍